.taskcluster.yml

```diff
@@ -694,3 +694,75 @@ tasks:
           dep check
       cache:
         generic-worker-checkout: /go/src
+
+
+  ##########################################################
+  ################ macOS amd64 Docker Build ################
+  ##########################################################
+
+  - provisionerId: pmoore-manual
```

Comment:
Will this only work for your iMac?

Reply:
No, anything I manully provisioner I've been giving `"provisionerId": "pmoore-manual"` - so this covers a few different [worker types](https://tools.taskcluster.net/provisioners/pmoore-manual/worker-types), that themselves cover a few different workers.

The `provisionerId` essentially just partitions worker types into distinct projects.

Reply:
Nice! Thanks for that link :)